tools/cli/internal/openapi/filter/code_sample.go

```diff
@@ -108,7 +131,8 @@ func (f *CodeSampleFilter) includeCodeSamplesForOperation(pathName, opMethod str
 	}
 
 	op.Extensions[codeSampleExtensionName] = []codeSample{
-		f.newCurlCodeSamplesForOperation(pathName, opMethod),
+		f.newServiceAccountCurlCodeSamplesForOperation(pathName, opMethod),
```

Comment:
product nit: can we make AtlasCLI first then Curl requests after?

Reply:
will do! but i think it's a no-brainer since curl is not a product for us, we could leave atlascli before curl and revert later based on product, no strong opinion though